docs/notebooks/tremor.ipynb

```diff
@@ -0,0 +1,82 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Tremor analysis"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
```

Comment:
Hier zie je een execution count staan: als je naar je notebook gaat, die je een getal (15) tussen vierkant [] haakjes staan. Dit is een counter voor hoeveel cellen je gedraaid hebt. Het is netjes om voor je push even _restart_ en _clear all outputs_ te klikken bovenin.